pkg/app/piped/notifier/slack.go

```diff
@@ -151,6 +152,19 @@ func (s *slack) buildSlackMessage(event model.NotificationEvent, webURL string)
 			{"Started At", makeSlackDate(d.CreatedAt), true},
 		}
 	}
+	generateDeploymentEventDataForTriggerFailed := func(app *model.Application, hash, msg string) {
+		var err error
+		link, err = git.MakeCommitURL(app.GitPath.Repo.Remote, hash)
```

Comment:
This function scope-defined `link` should be linked to the PipeCD application detailed page to unify with others. In case of link to the git commit, I think we should not shadow the scope-defined link value, defining kind of `commitURL` or `gitURL` and using it in L165 could be better.

Reply:
I'm middle of the road that which is better: link to the Application detail page or the specific commit page on Git hosting service.

Reply:
Well, for now make it be able to navigate to the Application details page, and then it's fine to refine as you like later.

Reply:
3956a08